### PR TITLE
Update docs about Keras models

### PR DESCRIPTION
**Pull request by Wauplin**
Related to https://github.com/keras-team/keras/pull/19854, https://github.com/huggingface/huggingface.js/pull/774 and https://github.com/huggingface/moon-landing/pull/10501 (private).

This PR updates the doc page about Keras to explain how to load/save models to the Hub using Keras nativly. For now we don't have proper examples yet but it's still better than the current outdated one. 

For review, see:
- https://moon-ci-docs.huggingface.co/docs/hub/pr_1320/en/keras
- https://moon-ci-docs.huggingface.co/docs/hub/pr_1320/en/tf-keras

### CODE REVIEW

File: docs/hub/keras.md [hunk 1]
```diff
@@ -1,75 +1,54 @@
...
-print(f'The image is a {classes[(np.argmax(prediction))]}!')
-
-# The image is a sunflower!
+model = keras.saving.load_model("hf://Wauplin/mnist_example")
 ```
 
-If you want to see how to load a specific model, you can click **Use in keras** and you will be given a working snippet that you can load it! 
-
-<div class="flex justify-center">
-<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-keras_snippet1.png"/>
```
Comment (julien-c) [inline docs/hub/keras.md:39]: keep some version of the existing one in a different (`tf-keras`?) page?

Comment (Wauplin) [reply docs/hub/keras.md]: right, I buried tf-keras a bit too quickly. I added some docs back in https://github.com/huggingface/hub-docs/pull/1320/commits/b9d3fb9cef4fb517135f94ec65d3eb650a5b076c and https://github.com/huggingface/hub-docs/pull/1320/commits/8146d787bfc72e0e09813fab9ed7518d0313baa6 as a section at the end of the Keras.md page, specially mentioning it's legacy and recommended to upload Keras3 models for better reach.

**Review by julien-c — APPROVED** [review]

File: docs/hub/keras.md (hunk 1 above)
Comment (julien-c) [reply docs/hub/keras.md]: i would have kept a separate page (if only because the library ids are ≠), but 🤷

Comment (Wauplin) [reply docs/hub/keras.md:39]: Make sense if we want to reference it. I moved it to a new TF-Keras page in https://github.com/huggingface/hub-docs/pull/1320/commits/f4707f44e67f50f987db0e3d446cf9d56d207236 and https://github.com/huggingface/hub-docs/pull/1320/commits/9df147e873f3db8bdb87cd86b13250fe511e278e.

**Review by julien-c — APPROVED** [review]

File: docs/hub/keras.md [hunk 2]
```diff
@@ -1,75 +1,49 @@
...
+model.save("hf://your-username/your-model-name")
 ```
-The repository will host your TensorBoard traces like below.
-
-<div class="flex justify-center">
-<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-keras_tensorboard.png"/>
-<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-keras_tensorboard-dark.png"/>
-</div>
 
+By default, the repository will contain a minimal model card. Check out the [Model Card guide](https://huggingface.co/docs/hub/model-cards) to learn more about model cards and how to complete them. You can also programmatically update model cards using `huggingface_hub.ModelCard` (see [guide](https://huggingface.co/docs/huggingface_hub/guides/model-cards)).
```
Comment (osanseviero) [inline docs/hub/keras.md:43]: Should we add something about keras format vs h5?

**Review by osanseviero — APPROVED** [review]
Nice!

File: docs/hub/keras.md (hunk 2 above)
Comment (Wauplin) [reply docs/hub/keras.md:45]: What do you think of https://github.com/huggingface/hub-docs/pull/1320/commits/2f1b6d58ed23b4c0c75e1bf0d06e13140383631c?

File: docs/hub/keras.md [hunk 3]
```diff
@@ -1,75 +1,56 @@
 # Using Keras at Hugging Face
 
-`keras` is an open-source machine learning library that uses a consistent and simple API to build models leveraging TensorFlow and its ecosystem.
+Keras is an open-source multi-backend deep learning framework, with support for JAX, TensorFlow, and PyTorch. You can find more details about it on [keras.io](https://keras.io/).
 
 ## Exploring Keras in the Hub
 
 You can find over 200 `keras` models by filtering at the left of the [models page](https://huggingface.co/models?library=keras&sort=downloads).
```
Comment (Vaibhavs10) [inline docs/hub/keras.md:7]:
```suggestion
You can find over 1200 `keras` models by filtering at the left of the [models page](https://huggingface.co/models?library=keras&sort=downloads).
```

smol suggestion: https://huggingface.co/models?library=keras&sort=downloads
(or we should change the tag if these are not the ones we want to showcase).

Comment (Wauplin) [reply docs/hub/keras.md:7]: Thanks @Vaibhavs10! Actually this is indeed the link we want to share but it is currently polluted by Keras 2.x models ([see this list](https://huggingface.co/models?library=keras,tf-keras&sort=downloads)). I'm in the process of opening PRs to fix these but maybe we just shouldn't provide a fixed number in the docs. I'll review this, thanks for spotting it!

Comment (Wauplin) [reply docs/hub/keras.md]: addressed in https://github.com/huggingface/hub-docs/pull/1320/commits/9a754c09df38fb8dc639ef123f14351a4454b0bc

**Pull request merged**